app/views/coronavirus_form/name.html.erb

```diff
@@ -15,6 +15,10 @@
   margin_top: 0
 } %>
 
+<%= render "govuk_publishing_components/components/hint", {
```

Comment:
Should this be inside the form_tag?

Reply:
Ah, I didn't copy 100% correctly from `app/views/contact_details.html.erb`. Yes, both the title and the hint are inside the form tag there. 😴One moment...

Reply:
Done in b932ce2.